action.yml

```diff
@@ -193,6 +187,31 @@ runs:
           echo "Set NIX_INSTALLER_ROOT_DISK=$NIX_INSTALLER_ROOT_DISK"
         fi
 
+        if [ -n "${{ inputs.nix-installer-pr }}" ] && [ -n "${{ inputs.nix-installer-tag }}" ]; then
```

Comment:
It seems like these also conflict with nix-installer-url, right? Since these are only reachable via the "else" branch of "if inputs.nix-installer-url".

Reply:
Oh, yes!